### ISSUE TITLE
If 2 jasmine tests have the same 'it' name, with the same 'describe' name, they are only counted once.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#365]
```
What steps will reproduce the problem?
1. Create test/JSTestDriverTest.js:
describe("JSTestDriver", function () {
  it("should be able to add numbers", function () {
    expect(1 + 1).toBe(3);
  });
  it("should be able to add numbers", function () {
    expect(2 * 2).toBe(4);
  });
  it("should be able to add numbers", function () {
    expect(1 + 1).toBe(3);
  });
  it("should be able to add numbers", function () {
    expect(2 * 2).toBe(4);
  });
});
2. Create a jsTestDriver.conf:
server: http://localhost:9876
load:
  - test/lib/jasmine/jasmine.js
  - test/lib/jasmine-jstd-adapter/JasmineAdapter.js
  - test/JSTestDriverTest.js

3. Run tests

What is the expected output? 
Total 4 tests (Passed: 2; Fails: 2; Errors: 0) // From the command line

What do you see instead?
Done: 1 of 1 Failed: 2 (See Screen Capture attached)

What version of the product are you using?
Version 117.222, 2012 April (build on JsTestDriver-1.3.4.b)
RubyMine 4.0.3

On what operating system?
Windows 7

Please provide any additional information below.
```

Original issue reported on code.google.com by `dan...@gmail.com` on 18 Apr 2012 at 3:13

Attachments:
- [Capture.PNG](https://storage.googleapis.com/google-code-attachments/js-test-driver/issue-365/comment-0/Capture.PNG)

**Comment by GoogleCodeExporter** [comment]
Original comment by `Sergey.S...@gmail.com` on 20 Apr 2012 at 4:00
- Changed state: **Accepted**
- Added labels: **IntelliJ**

**Comment by GoogleCodeExporter** [comment]
```
Issue 362 has been merged into this issue.
```

Original comment by `Sergey.S...@gmail.com` on 20 Apr 2012 at 4:01

**Comment by GoogleCodeExporter** [comment]
```
Fixed in RubyMine 4.5.
```

Original comment by `Sergey.S...@gmail.com` on 21 Jul 2012 at 6:59
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Issue 362 has been merged into this issue.
```

Original comment by `Sergey.S...@gmail.com` on 11 Sep 2012 at 2:05
